lib/node_modules/@stdlib/math/base/special/cosd/package.json

```
@@ -67,5 +67,83 @@
     "cosine",
     "trig",
     "trigonometry"
-  ]
+  ],
+  "__stdlib__": {
+    "scaffold": {
+      "$schema": "math/base@v1.0",
+      "base_alias": "cos",
+      "alias": "cosd",
+      "pkg_desc": "compute the cosine of a double-precision floating-point number (in degrees).",
+      "desc": "computes the cosine of a double-precision floating-point number (in degrees)",
```

```suggestion
      "pkg_desc": "compute the cosine (in degrees)",
      "desc": "computes the cosine (in degrees)",
```

I am now realizing that these descriptions need to generalize a bit more beyond a "number" given how this scalar kernel will be consumed in downstream contexts.